code/index.html

```diff
@@ -3,28 +3,190 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Cordonnerie</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Oswald:wght@700&family=PT+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/e056cb308d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="shortcut icon" href="./img/favicon.ico" type="image/x-icon" />
+    <link rel="icon" href="./img/favicon.ico" type="image/x-icon" />
   </head>
+
   <body>
+    <!-- <header> -->
+    <!-- Sticky navbar -->
+    <nav class="navbar">
+      <a href="#"
+        ><img src="./img/logo.png" alt="" width="80px" height="80px"
+      /></a>
+      <ul>
+        <li>
+          <a href="#">Home</a>
+        </li>
+        <li>
+          <a href="#">Shoe care</a>
+        </li>
+        <li>
+          <a href="#">FAQ</a>
+        </li>
+        <li>
+          <a href="#">About us</a>
+        </li>
+      </ul>
+    </nav>
+
     <!-- video or image as a header is cool :) -->
+    <section class="hero">
+      <video loop muted autoplay preload="auto" style="pointer-events: none">
+        <source src="./videos/video.mp4" type="video/mp4" />
+        Your browser does not support the video tag.
+      </video>
+      <div class="hero-text-container">
+        <h1>Cordonnerie</h1>
+        <h2>we take care of your shoes</h2>
+      </div>
+    </section>
+    <!-- </header> -->
 
-    <!-- Signup form -->
+    <!-- Main content -->
+    <section class="main">
+      <section class="main-section information">
+        <p><strong>Cordonnerie</strong> - your online cobbler</p>
+        <p>For fast and professional repair of all kinds of shoes</p>
+      </section>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+      <!-- Signup form -->
+      <form
+        class="main-section form"
+        action="http://httpbin.org/anything"
+        method="POST"
+      >
+        <p>Create your account to get started:</p>
+        <div class="form-item">
+          <!-- <label for="email">Email</label> -->
+          <input id="email" type="email" name="email" placeholder="Email" />
+        </div>
+        <div class="form-item">
+          <!-- <label for="name">Password</label> -->
+          <input
+            id="password"
+            type="password"
+            name="password"
+            placeholder="Password"
+            minlength="8"
+          />
```

Comment:
Nice to see some simple validation 🌟 If you don't want to submit empty input, you can check this out:
https://www.w3schools.com/tags/att_input_required.asp